jobs/kube-proxy-windows/templates/config/config.yml.erb

```diff
@@ -11,3 +11,7 @@
 %>
 <% require 'yaml' %><%= p('kube-proxy-configuration').to_yaml %>
 hostnameOverride: <%= hostname %>
+<% if link('flanneld-windows').p('backend-type') == "win-overlay" %>
```

Comment:
This should be passed through the manifest when enabling `win-overlay`, not getting consumed from the link.

This brings it in-line with our desire to have most (if not all) configuration stem from the deployment manifest, not from random computation determined at templating time.